### DISCUSSION TITLE
Get Mods

### DISCUSSION THREAD

**Discussion by LMH01** [LMH01/MGT2_Mod_Tool#34] [Show and tell]
### Installation:
To install mods you have found here you just need to do the following:
1. Download the mod tool (At least version 4.1.0): [link](https://github.com/LMH01/MGT2_Mod_Tool/releases/tag/v4.1.0)
2. Copy the url
3. Open the mod tool and navigate to: `Mods -> Import -> Import from URL`
4. Paste the url into the text field and click `OK`
5. If you like you can customize what mods should be imported. For that you can enable the checkbox `Customize import`.
6. To start the import klick `yes`
7. If everything worked a message will say that everything went properly
8. Start the game and enjoy the mods

### Submit your own mods:
If you made your own mod pack and would like to share it with others you can just write a comment under this thread or message me on discord (My username is LMH01). I will then check out the mod and i might add it to the get mods list below.

#### LMH01 Beginner pack by LMH01
This modpack contains the following:
4 new genres: Horror, Survival, Sandbox, Metroidvania
7 new publishers, at least one for each new genre
7 new npc ip's, one for each new publisher
2 new licences
Download: https://www.dropbox.com/s/405iorph2z2d25c/LMH01_Beginner_pack.zip?dl=1

#### Raytracing Paket by Loyalathletics [Incompatible scince v3.0.0]
This modpack contains the following:
3 new Engine Features
Download: [https://docs.google.com/uc?export=download&id=1LszbK02k7rCHCbmCvWbeK7bLivjVvFG_](https://docs.google.com/uc?export=download&id=1LszbK02k7rCHCbmCvWbeK7bLivjVvFG_)

#### Engine and Publisher Extension [MOD] by chefkoch [Incompatible scince v4.0.0]
This modpack contains the following:
8 Engine Features
7 Publisher
1 Developer
8 Topics
5 Licences
Additional information can be found here: [link](https://steamcommunity.com/app/1342330/discussions/1/3101264555060468215/)
Download: https://www.dropbox.com/s/xoh1vue4k3nlxdn/Engine%20and%20Publisher%20Extension%20%5BMOD%5D%20by%20chefkoch%20%28for%20v3.0%29.zip?dl=1

#### Harry Potter Movie Licenses by Cvexreos [Incompatible scince v4.2.0]
This modpack contains the following:
8 Licences - One for each Harry Potter movie
Download: https://www.dropbox.com/s/szau65qufjl9hi6/MGT2_Harry_Potter_Movie_Licences%20by%20Cvexreos%20%28for%20v3.0%29.zip?dl=1

**Comment by Joti17** [comment]
Kannst du eine Ultra op Version veröffentlichen, wo es keine limets gibt, und ich die zahl bei z.B. Tech reinschreiben kann und nicht die ganze Zeit denn hoch knopf drücken muss
